backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -142,7 +140,7 @@ class EakApplicationMutationService {
                     it.tryChangeStatus(Status.Approved)
                     ApplicationView.fromDbEntity(it, true)
                 }
-                ?: throw NotFoundException("Application not found")
```

Comment:
for NotFoundException we show "Seite nicht gefunden" error message in administration, I believe it's not relevant for this case